src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/removeConfirm.jsx

```diff
@@ -5,24 +5,17 @@ import ConfirmHeader from 'app/views/settings/account/accountSecurity/components
 import Confirm from 'app/components/confirm';
 import TextBlock from 'app/views/settings/components/text/textBlock';
 
-class RemoveConfirm extends React.Component {
-  render() {
-    return (
-      <Confirm
-        message={
-          <React.Fragment>
-            <ConfirmHeader>{t('Do you want to remove this method?')}</ConfirmHeader>
-            <TextBlock>
-              {t(
-                'Removing the last authentication method will disable two-factor authentication completely.'
-              )}
-            </TextBlock>
-          </React.Fragment>
-        }
-        {...this.props}
-      />
-    );
-  }
-}
+const message = (
```

Comment:
What's the reason for pulling this out of the component?

Performance is neglible either way, but seems like due to the lower frequency of use of this component, we should inline it in `RemoveConfirm` so that it's lazily evald?

(No changes required here, just a point of discussion)

Reply:
That's a good point. I pulled it out for stylistic reasons. But you make a good point. I'll change in a diff PR so I can merge this now.